src/PerfView/SupportFiles/UsersGuide.htm

```diff
@@ -2295,6 +2295,10 @@ <h2>
                     views that are more specialize investigations that are rarer than the common CPU,
                     wall clock time, and memory investigations.
                     <ul>
+                        <li>
+                            <strong><a id="ProcessorStacks">Processor Stacks</a></strong> - Shows what every
+                            processor is doing, including at what priority. Use the 
```

Comment:
Looks like you started this, but didn't finish :)

Reply:
Yeah :) Added that GroupPats can be used to focus on processor or priority.

Reply:
Awesome.  Thanks!